### PR TITLE
addons/prometheus: update grafana dashboard

### PR DESCRIPTION
**Pull request by aanm**
It contains the recently added `bpf_maps_virtual_memory_max_bytes`
and `bpf_progs_virtual_memory_max_bytes` metrics.

Signed-off-by: André Martins <andre@cilium.io>

```release-note
Updated grafana dashboard
```

### CODE REVIEW

**Comment by errordeveloper** [comment]
Is this old manifest being converted to a chart?

File: examples/kubernetes/addons/prometheus/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: prometheus
+description: Helm chart to deploy a simple prometheus and grafana pods
```
Comment (errordeveloper) [inline examples/kubernetes/addons/prometheus/Chart.yaml]:
```suggestion
description: A simple Helm chart to deploy Prometheus and Grafana with Cilium dashboard
```

**Comment by aanm** [comment]
>Is this old manifest being converted to a chart?

Not quite, it's just easier to generate the example manifest if it is based on a helm chart.

**Comment by aanm** [comment]
test-me-please

**Review by tklauser — APPROVED** [review]

**Pull request merged**